test/transcoding/SPV_INTEL_fpga_loop_controls/FPGALoopAttr.ll

```diff
@@ -186,12 +276,21 @@ attributes #0 = { convergent noinline nounwind optnone "correctly-rounded-divide
 !9 = distinct !{!9, !10}
 !10 = !{!"llvm.loop.max_concurrency.count", i32 2}
 !11 = distinct !{!11, !8, !10}
+!12 = distinct !{!12, !13, !14}
+!13 = !{!"llvm.loop.mustprogress"}
+!14 = !{!"llvm.loop.intel.loopcount_avg", i64 12}
+!15 = distinct !{!15, !13, !16, !17, !18}
+!16 = !{!"llvm.loop.intel.loopcount_min", i64 4}
+!17 = !{!"llvm.loop.intel.loopcount_max", i64 4294967396}
```

Comment:
A comment that would tie this number to a hexadecimal/binary counterpart would be appreciated.

Reply:
Added comment in this commit: [a7a3741](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1062/commits/a7a3741d55a23ee5bb5298c087fc5eaf778edda5)